### DISCUSSION TITLE
Is there a way to get the raw data of the Force Directed Graph from the System Architecture tab in jaeger-ui

### DISCUSSION THREAD

**Discussion by vxe** [jaegertracing/jaeger#3518] [Q&A]
New user, trying to get this data for a side project, is there a way to fetch the graph points as an aggregate somehow or is it all baked into the ui implementation?

thanks!

**Comment by vxe** [comment]
![image](https://user-images.githubusercontent.com/4325947/152627462-1c775d55-5e4e-472c-8954-190229e5df52.png)

specifically

**Reply by yurishkuro** [reply]
Of course, UI just loads the data as JSON from the /api/ endpoint. You can view the request and the data in devtools in Chrome.

**Comment by vxe** [comment]
found it thanks:

https://www.jaegertracing.io/docs/1.30/apis/#service-dependencies-graph-internal
